src/components/Main/Controls/index.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import Loader from "../Loader";
+import SingleControl from "./singleControl";
+
+function Controls({
+  ingredients,
+  quantities,
+  updateBurger,
+  clearAll,
+  loading,
+}) {
+  return (
+    <Wrapper>
+      {loading ? (
+        <Loader />
+      ) : (
+        <Div>
```

Comment:
Better to name custom components in another way. So the namespaces are not poluted

```suggestion
        <Container>
```